assets/dev/js/editor/regions/panel/pages/elements/favorites/commands/base.js

```diff
@@ -0,0 +1,71 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export default class CommandsBase extends CommandBase {
+	/**
+	 * @inheritDoc
+	 */
+	initialize( args = {} ) {
+		args = _.assign( { store: true }, args );
```

Comment:
Yes, those are default args. I'll probably just add a 'default' property to the component prototype so it's easier to access. I thought '_.' is the standard because it used everywhere here. Generally, the spread operator is more intuitive for me, so I guess I'll use it here.

Reply:
> can it be replaced by `Object.assign({ store: true}, args)` or `{ ...args, store: true }`?
> 
> is it a default arg? if yes please write an explanation comment.

Yes, those are default args. I'll probably just add a 'default' property to the component prototype so it's easier to access. I thought '_.' is the standard because it used everywhere here. Generally, the spread operator is more intuitive for me, so I guess I'll use it here.